packages/@lwc/integration-karma/test/rendering/slotting/x/regressionSlot/regressionSlot.js

```diff
@@ -6,7 +6,7 @@ export default class FocusTrap extends LightningElement {
     handleContentChange() {
         // trigger rehydration.
         // triggering the rehydration on the next tick throws.
-        Promise.resolve().then(() => this.counter++);
+        void Promise.resolve().then(() => this.counter++);
```

Comment:
Another `void`?

Reply:
See https://github.com/salesforce/lwc/pull/3995#discussion_r1489824341 for an explanation. Same rationale applies here (it's also a test file).